pkgs/applications/science/medicine/aliza/default.nix

```diff
@@ -0,0 +1,49 @@
+{ stdenv, fetchurl, dpkg, makeWrapper, patchelf, qt4, zlib, libX11, libXt }:
+
+with stdenv.lib;
+stdenv.mkDerivation {
+  name = "aliza";
+  src = fetchurl {
+    url = "https://drive.google.com/uc?export=download&id=0ByBLJ2-U1t9xZ2JUSFYyWTR1UFU";
+    sha256 = "16ysacpdf673sh6jgg52sc32a416lk2m227swkaw7l9yiryriyhb";
+    name = "aliza.deb";
+  };
+
+  buildInputs = [ dpkg makeWrapper ];
+
+  unpackCmd = "dpkg -x $curSrc deb";
+
+  patchPhase = ''
+    sed -i 's/^Exec.*$/Exec=aliza %F/' usr/share/applications/aliza.desktop
+  '';
+
+  installPhase = ''
+    mkdir -p $out
+    cp -r usr/bin usr/share $out
+
+    runHook postInstall
+  '';
+
+  postInstall = ''
+    ${patchelf}/bin/patchelf \
+      --interpreter "$(cat $NIX_CC/nix-support/dynamic-linker)" \
+      $out/bin/aliza
+
+    ${patchelf}/bin/patchelf \
+      --interpreter "$(cat $NIX_CC/nix-support/dynamic-linker)" \
+      $out/bin/aliza-vtkvol
+
+    wrapProgram $out/bin/aliza \
+      --prefix LD_LIBRARY_PATH : ${stdenv.lib.makeLibraryPath [ qt4 zlib stdenv.cc.cc ]}
+
+    wrapProgram $out/bin/aliza-vtkvol \
+      --prefix LD_LIBRARY_PATH : ${stdenv.lib.makeLibraryPath [ qt4 zlib stdenv.cc.cc libX11 libXt ]}
```

Comment:
You can use `--set-rpath` of patchelf instead of `LD_LIBRARY_PATH` to set the library path.